lib/views/alaveteli_pro/batch_request_authority_searches/_begginer_batch_pop_up.html.erb

```diff
@@ -0,0 +1,53 @@
+<% content_for :javascript_head do %>
+  <%= javascript_importmap_tags("modal", importmap: Rails.application.theme_importmap) %>
+<% end %>
+
+<div class="modal-content modal-hidden batch-request-pop-up"  aria-hidden="true" data-auto-open="true" data-open-delay="100" data-click-outside-close="false">
+  <div class="modal-container">
+    <div class="js-carousel" role="region" aria-label="Featured content" aria-roledescription="carousel" carousel-current-slide="0">
+      <div class="js-carousel__wrapper">
+        <div class="js-carousel__item" js-carousel-order="0" aria-roledescription="slide" aria-label="Slide 1 of 2">
+          <div class="modal-header">
+            <h2 class="modal-title">Responsible use of Batch</h2>
+          </div>
```

Comment:
We should also add an intro paragraph here along the lines of "hey you're new to batch… we trust you'll be a good user; here's a few important things before you continue"